research/flamby/README.md

```diff
@@ -17,8 +17,10 @@ cd <fl4health_repository>
 pip install --upgrade pip poetry
 poetry install --with "dev, dev-local, test, codestyle"
 cd <FLamby_repository>
+pip install albumentations==1.4.20
```

Comment:
Shouldn't we add this to poetry's `pyproject.toml` instead?

Reply:
Good question. This is specific to the FLamby environment. We don't use albumentations in our library but FLamby does in theirs. There is an issue using the most recent version of the library though. So this forces a pin while still allowing the rest of FLamby's dependencies to be installed

Reply:
I see. Is that the only dependency that is in flamby but not in fl4health?

Reply:
No. Unfortunately there are a lot more in FLamby that are necessary to support the different datasets.

Reply:
I see, I was just wondering if there was a requirements file somewhere that you could put this but looking into it here it seems like there isn't, so all good.

Reply:
Your thought process is a good one. We actually used to have a requirements file for installing the requirements of FL4Health and FLamby together. The problem was that it ends up out of date fairly quickly, as it's not tied to our poetry or their repository. If you have better suggestions for maintaining the compatibility as a follow up PR, I'm open to it 🙂